fdmi/plugins/m0scheduler

```diff
@@ -17,128 +17,277 @@
 # For any questions about this software or licensing,
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
+
+# This is FDMI plugin sample app wrapper that serves as a scheduler
+# for the FDMI records. It gathers that cluster infp using Hare hctl
```

Comment:
s/It gathers that cluster infp using Hare hctl/It gathers the cluster info using Hare `hctl`

Reply:
agreed